src/neural/onnx/converter.h

```diff
@@ -44,6 +44,7 @@ struct WeightsToOnnxConverterOptions {
   int batch_size = -1;
   int opset = 17;
   bool alt_mish = false;
+  bool alt_ln = false;
```

Comment:
Maybe rename it so something slightly less cryptic? (`alternative_layer_normalization` or `alt_layer_norm` or at the very least add a comment). And I'd say the same with the command line option.

Reply:
I agree, fixed